### ISSUE TITLE
Hovering and screen reader awareness

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#266444]
Type: <b>Feature Request</b>

It appears as though many places are adopting a hover state to show critical information or at least advanced information that a screen reader does not have access to as far as reading the text. Can this be populated somewhere for screen reader users to utilize such as the status bar, a command to hear the information, accessible view, or some other command that would render the hover information? ctrl+k plus ctrl+i is a command to show hover inbformation as I understand it but it seemingly does nothing for me, for example in the model picker.

VS Code version: Code 1.104.0 (f220831ea2d946c0dcb0f3eaa480eb435a2c1260, 2025-09-10T06:46:18.035Z)
OS version: Windows_NT x64 10.0.26100
Modes:

**Comment by meganrogge** [comment]
Hi, thanks for the issue! We've fixed the model picker issue by including the hover content in the aria label directly (this makes more sense anyway as without hovering, sighted users see that info). I'm not aware of other contexts where this does not work for screen reader users. Please create an issue if you have noticed other areas that need attention.
